<launch>
  <arg name="arena_info_static_topic" value="/arena_info_static" />
  <arg name="arena_info_dynamic_topic" value="/arena_info_dynamic" />
  <arg name="ctrl_topic" value="/ctrl/agent_0" />

  <arg name="playground" value = "highway_lite" />

  <node pkg="planning_integrated" type="test_ssc_with_mpdm" name="test_ssc_with_mpdm_0" output="screen">
    <param name="ego_id" type="int" value="0" />
    <param name="agent_config_path" type="string" value="$(find playgrounds)/$(arg playground)/agent_config.json" />
    <param name="ssc_config_path" type="string" value="$(find ssc_planner)/config/ssc_config.pb.txt" />
    <param name="desired_vel" value="20.0"/>
    <param name="use_sim_state" value="true"/>
    <remap from="~arena_info_static" to="$(arg arena_info_static_topic)"/>
    <remap from="~arena_info_dynamic" to="$(arg arena_info_dynamic_topic)"/>
    <remap from="~ctrl" to="$(arg ctrl_topic)"/>
  </node>
</launch>
